### PR TITLE
Use errors.New() to output the error message and fix some typos

### PR DESCRIPTION
**Pull request by fate-grand-order**
Use errors.New() directly to output the error message and fix some typos I found.

Signed-off-by: fate-grand-order <chenjg@harmonycloud.cn>

### CODE REVIEW

File: configuration/configuration.go [hunk 1]
```diff
@@ -474,7 +475,7 @@ func (storage Storage) MarshalYAML() (interface{}, error) {
 // Auth defines the configuration for registry authorization.
 type Auth map[string]Parameters
 
-// Type returns the auth type, such as htpasswd or token
+// Type returns the auth type, such as passwd or token
```
Comment (Windfarer) [inline configuration/configuration.go]: this is not a typo!

Comment (fate-grand-order) [inline configuration/configuration.go]: oh, I'm so sorry! I will update it.

File: errors.go [hunk 2]
```diff
@@ -77,7 +77,7 @@ func (err ErrManifestUnknownRevision) Error() string {
 type ErrManifestUnverified struct{}
 
 func (ErrManifestUnverified) Error() string {
-	return fmt.Sprintf("unverified manifest")
+	return fmt.Sprint("unverified manifest")
```
Comment (dmcgowan) [inline errors.go]: just `return "unverified manifest"`

Comment (fate-grand-order) [inline errors.go]: ok, the PR has updated. @dmcgowan

**Comment by dmcgowan** [comment]
LGTM

**Pull request merged**